src/panels/scene-viewer/module.ts

```diff
@@ -46,5 +46,12 @@ export const plugin = new PanelPlugin<PanelOptions>(ScenePanel)
         settings: {
           options: getVariableOptions(),
         },
+      })
+      .addSelect({
+        path: 'customInputActiveCamera',
+        name: 'Default Active Camera',
```

Comment:
better to be `Active camera variable name`? since it's the template variable name not the actual camera

Reply:
NOOOOOOO!!!!!!! lol Ok

Reply:
Done